### ISSUE TITLE
v1.0.0 - Tracking Issue

### ISSUE THREAD

**Issue by cecilia-sanare** [rain-cafe/protontweaks#6]
This is a overarching list of all the necessary tasks to reach release candidate for v1.0.0

### Issues

- Update https://protontweaks.com to be a splash page about the CLI with a link to the Database
- Pop up a ui to indicate to users that something is being applied. Similar to how https://github.com/simons-public/protonfixes does it
- Convert all of the existing code to run async and run tasks in parallel where possible.
- Add more install options. `AppImage`, `cargo install`, other prebuilt binaries, etc.

### Stretch Goals

- Setup a repository for easily keeping binaries up to date (v1+)
- Look into improving the performance of installing tweaks
